gini/gini-capture-sdk-android
# Bugfix/pia 1425 gracefully fail if no singleton

If the `GiniCapture` instance is missing then a `MISSING_GINI_CAPTURE_INSTANCE` error is returned.

In the Screen API the error is returned as an activity result and in the Component API it's returned in the `onError()` listener method of the fragment which noticed the missing instance.

### How to test
Do the following for each of the screens listed in the [ticket's acceptance criteria](https://ginis.atlassian.net/browse/PIA-1425):
* Navigate to the screen.
* Put the app in background.
* Simulate system kill: run `adb shell am kill --user current net.gini.android.capture.screenapiexample` for the Screen API example app or `adb shell am kill --user current net.gini.android.capture.componentapiexample` for the Component API example app.
* Bring the app back to the foreground.
* A message should show that the `MISSING_GINI_CAPTURE_INSTANCE` error occurred.

### Ticket 
https://ginis.atlassian.net/browse/PIA-1425
